.github/workflows/docker.yml

```
@@ -0,0 +1,81 @@
+name: Create and publish a Docker images
+
+on:
+  push:
+    tags:
+      - "v*.*.*"
+    branches:
+      - "master"
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
```

```suggestion
  build-and-push-image:
    name: PHP ${{ matrix.php-version }}, Fixer ${{ github.ref == 'refs/heads/master' && 'nightly' || ${{ github.ref_name }} }}
```

I believe we can omit Alpine version in the job's name since it's a technical detail that is not that important at the overview level. Also "docker build" in the jobs' names would be redundant since workflow's name contain this information already (we have something similar in [CI](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/actions/runs/7263207097) workflow). The intent here is to have friendly job names, depending on which triggered the workflow it would be `Docker build PHP 8.3, Fixer nightly` for `master` branch, or `Docker build PHP 8.3, Fixer v3.40.0` for tags.

PS. "nightly" or "edge", whatever. I just did not want to have `Fixer master` because it would look weird 😅.